lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/dataaccess/DBOSubmissionDAOImplTest.java

```diff
@@ -281,9 +314,74 @@ public void testListSubmissions() {
 				false, 10L, 0L);
 		assertNotNull(submissions);
 		assertEquals(0, submissions.size());
-
-		submissionDao.delete(dto1.getId());
-		submissionDao.delete(dto2.getId());
+	}
+	
+	private static SubmissionInfo createSubmissionInfo(ResearchProject rp, long modifiedOn) {
+		SubmissionInfo result = new SubmissionInfo();
+		result.setInstitution(rp.getInstitution());
+		result.setIntendedDataUseStatement(rp.getIntendedDataUseStatement());
+		result.setProjectLead(rp.getProjectLead());
+		result.setModifiedOn(new Date(modifiedOn));
+		return result;
+		
+	}
+	
+	@Test
+	public void testListInfoForApprovedSubmissions() {
```

Comment:
I need to take some notes to understand this test, let's spend extra time on this section during our meeting this afternoon.

Reply:
Yes, this is the crucial part of the code review.